data/hazards.txt

```diff
@@ -0,0 +1,75 @@
+# Copyright (c) 2020 by Zitchas
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+hazard "EW Base Heat"
+	"duration" 1 200
+	"strength" 0.25 2
+	"range" 0 25000
+	"environmental effect" "heat hazard" 300
+	weapon
+		"relative heat damage" 0.0005
+
+
+
+effect "heat hazard"
+	sprite "effect/ew heat/ew heat"
+		"frame rate" 8
+		"random start frame"
+		"rewind"
+	"lifetime" 60
+	"velocity scale" 3
+
+
+
+hazard "EW Base Storm"
+	"duration" 300 1500
+	"strength" 0.5 3
+	"range" 0 25000
+	"environmental effect" "ion hazard" 300
+	weapon
+		"ion damage" 1
+		"shield damage" 5
+
+
+
+effect "ion hazard"
+	sprite "effect/spark"
+		"frame rate" 30
+		"random start frame"
+	"lifetime" 60
+	"random angle" 360
+	"random spin" 10
+	"random velocity" 1
+
+
+
+hazard "Grav Strong"
+	"constant strength"
+	"range" 0 2500
+	weapon
+		"gravitational"
+		"hit force" -25
+		"damage dropoff" 100 2500
+		"dropoff modifier" 0
+
+
+
+hazard "Grav Deadly"
+	"constant strength"
+	"period" 10
+	"strength" 0.1 2
+	"range" 0 150
+	weapon
+		"hit effect" "nano spark"
+		"shield damage" 50
+		"hull damage" 50
+		"damage dropoff" 0 150
+		"dropoff modifier" 0
+		"piercing" 0.25
```

Comment:
How about using a relative hull strength here? These values mean that any ship that falls too close to the black hole will be dead after a minute.
```suggestion
	"constant strength"
	"range" 150
	weapon
		"hit effect" "nano spark"
		"damage dropoff" 0 150
		"dropoff modifier" 0
		"relative hull damage" .00028
		"piercing" 10
```
And I believe this may have been mentioned on Discord, but how about another hazard, "Black Hole Accretion Disk", that extends to the edge of the accretion disk graphic and deals shield damage instead of piercing hull damage, giving the player a warning zone before they actually start to die as they fall into the black hole.